docs/src/edit-link.js

```diff
@@ -2,27 +2,51 @@ import React from 'react'
 import { Location } from '@reach/router'
 import { css } from 'theme-ui'
 
-const base = 'https://github.com/styled-system/styled-system/edit/master/docs'
+// Removes preceding and trailing slashes
+const parsePage = string => string.replace(/\/+$/, '').substr(1)
 
-const getHREF = location => {
-  if (location.pathname === '/') return base + '/getting-started.md'
-  return base + location.pathname.replace(/\/+$/, '') + '.md'
+const base = 'https://github.com/styled-system/styled-system/edit/master'
+
+// Pages that don't reference the default /docs/{page}.md
+const custom = {
+  css: '/packages/css/README.md',
+  demo: '/docs/src/pages/demo.js',
+}
+
+// Don't display the edit link on these pages:
+const ignore = ['babel-plugin']
```

Comment:
Thanks for fixing this – it's something I forgot about! I sort of think we could remove the Babel plugin from the docs entirely (separate from this PR) since it *is* experimental and hasn't really been vetted much

Reply:
Might it be worth leaving in but making a clear note that it is experimental?